music_assistant/server/providers/plex/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "music",
+  "domain": "plex",
+  "name": "Plex Media Server Library",
+  "description": "Support for the Plex streaming provider in Music Assistant.",
+  "codeowners": ["@micha91"],
+  "requirements": ["plexapi==4.13.2", "async-lru==2.0.2"],
```

Comment:
as discussed, let's have a look if we should switch to async_lru completely or adjust your implementation to MA cache, I'd rather not have 2 cache implementations. That said,  maybe you can even skip the cache completely if you could use ItemMapping objects ?